AUTHORS.md

```diff
@@ -1,277 +1,150 @@
+---
+orphan: true
+---
 
-Authors
-========
+# Authors
 
-Editors
---------
-
-- K\. Jarrod Millman
+## Editors
 
+- K. Jarrod Millman
 - Stéfan van der Walt
-
 - Gaël Varoquaux
-
 - Emmanuelle Gouillart
-
 - Olav Vahtras
-
 - Pierre de Buyl
+- Peter Rush
+- Matthew Brett
 
-
-Chapter authors
-----------------
+## Chapter authors
 
 Listed by alphabetical order.
 
 - Christopher Burns
-
 - Adrian Chauve
-
 - Robert Cimrman
-
 - Christophe Combelles
-
 - André Espaze
-
 - Emmanuelle Gouillart
-
 - Mike Müller
-
 - Fabian Pedregosa
-
 - Didrik Pinte
-
 - Nicolas Rougier
-
 - Gaël Varoquaux
-
 - Pauli Virtanen
-
 - Zbigniew Jędrzejewski-Szmek
-
 - Valentin Haenel (editor from 2011 to 2015)
 
-Additional Contributions
-------------------------
+## Additional Contributions
```

Comment:
This is separate from the PR but as the diff is here I comment on it: we should automate this list somehow and maybe use `.mailmap` so clean up duplicates?